tada_a_bigoh.py

```diff
@@ -83,9 +95,29 @@
             total_loop_list.append(current_benchmark.get_total_loops())
             # perform additional analysis of the results
             # reminder: print('Values {0}'.format(current_benchmark.get_values()))
-            mean = current_benchmark.mean()
+            if tada_arguments.types[0] == "hypothesis-sub":
+                # with open("data.txt", "r") as f:
+                #     hypothesis_runtime = float(f.read())
+                tada_configuration_dict = configuration.read(constants.CONFIGURATION)
+                hypothesis_gen_start = time.time()
+                temp_analyzed_function = generate.generate_strategy(
+                    generate.time_generation,
+                    configuration.get_schema_path(tada_configuration_dict),
+                    current_size,
+                )
+                i = 0
+                total_gen_time = 0
+                while i < 11:
+                    hypothesis_gen_start = time.time()
+                    temp_analyzed_function()
+                    total_gen_time += time.time() - hypothesis_gen_start
+                    i += 1
+                constants.GEN_TIME = total_gen_time / 10
+            # generate.store_hypothesis(constants.GEN_TIME)
+            mean = current_benchmark.mean() - constants.GEN_TIME
+            print("Hypothesis Generation Time: ", constants.GEN_TIME)
```

Comment:
`hypothesis-sub` shouldn't be working and added at this point since there is no data generation strategy that still includes data generation time. Subtracting this time is just subtracting time from the experiment itself, right?

Reply:
You are right. I forgot that `perfbenchmark` was changed previously. I will fix this problem this afternoon. Nice catch.

Reply:
fixed

Reply:
removed